reconcile/fleet_labeler/integration.py

```diff
@@ -99,6 +99,16 @@ def reconcile(self, dependencies: Dependencies) -> None:
                 ocm=ocm,
                 dry_run=synch_labels,
             )
+            num_labels = 0
+            for cluster in spec.clusters:
+                all_cluster_ids.add(cluster.cluster_id)
+                num_labels += sum(1 for _ in cluster.subscription_labels)
```

Comment:
is `cluster.subscription_labels` a `dict`? why not directly `num_labels += len(cluster.subscription_labels)`

Reply:
Absolutely! Fixed